integration-test.ts

```diff
@@ -116,6 +119,72 @@ async function handleMission(mission: Mission<MissionParams>) {
     }
 }
 
+async function startMission(mission: Mission<MissionParams>) {
+    try {
+        const conf = {
+            ...config,
+            "ethNodeUrl": wallet.nodeUrl,
+            "blockchainType": BlockchainType.test
+        }
+        // Init web3 object
+        const web3 = Contracts.initWeb3(conf);
+
+        // Get contract instance and address
+        const { contract, contractAddress } = Contracts.getContract(ContractTypes.basicMission, web3, conf);
+
+        const walletPublicKey = wallet.address;
```

Comment:
Is that really the public key or the address? or are these the same?

Reply:
They are the same (`walletPublicKey === wallet.address`).